frontend/src/App.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { server, rest, within, render, screen } from './test-utils'
+import App from 'App'
+
+test('renders with no errors', async () => {
```

Comment:
This would probably be a better test if:
1. More specific about the behaviour you're testing for. E.g. should show landing page when route is `/`.
2. Try to test for only one thing per test. So it will be better to have multiple tests for each expected route, rather than one test for all routes

The general idea is that if the test fails, you should be able to quickly pinpoint what broke. In this case, if `renders with no errors` failed, it's not immediately obvious what needs to be fixed.

Reply:
Thanks! I originally included this test just to ensure that tests run fine on Travis and Amplify, so I didn't put much thought into the naming. I've renamed the test to "displays landing page when unauthenticated".

About having multiple tests, I feel like there isn't really a reason to do that for this test. I think this test is simple enough to be able to pinpoint errors quickly from the console error. Furthermore, having to render the route multiple times would increase the overall running time of the test suite.

I will keep these in mind for future tests though!